fpsim/locations/location_template.py

```diff
@@ -0,0 +1,826 @@
+# -*- coding: utf-8 -*-
+"""
+FPsim shell for calibrating new contexts - use context-specific data to fill in
+This script sets the parameters for a new location
+Save a copy, name the copy 'location'.py (e.g. ethiopia.py; bihar.py)
+"""
```

Comment:
I think it would be helpful to have a line of instructions something like: "Location specific parameters are commented out. Fill in values for your location then remove the comment."